build-native-with-native.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+export FLAVOR=NATIVE_WITH_NATIVE
```

Comment:
it looks like it makes sense to combine logic with 
build-native-with-cross.sh

Reply:
What do you mean to share in particular? Most of the logic is shared through `build-package.sh` script. The rest serves mostly as documentation of the differences of the build processes.

Reply:
build-native-with-native.sh and build-native-with-cross.sh look almost identical.

Reply:
I can create a new script called `build-native-shared.sh`, hidden in `.github/scripts/` folder and call that script with proper setup from `build-native-with-native.sh` and `build-native-with-cross.sh`, here in to root of the repository. This would obfuscate the build process for the readers though. Is that aligned with your intention?

Reply:
I would rename it to build-native.sh and use FLAVOR configuration. It might be useful to use also config.sh.

Reply:
OK, I'll implement that suggestion within https://github.com/Windows-on-ARM-Experiments/msys2-woarm64-build/pull/19/files PR as here it would be already to time consuming when that PR exist.